### PR TITLE
Add util for strings suffix check

### PR DESCRIPTION
**Pull request by pankrator**
# Pull Request Template

## Motivation
Needed for suffix checks

## Pull Request status

* [x] Initial implementation

### CODE REVIEW

**Comment by dpanayotov** [comment]
Why is this needed? Do you import it in some other project? If so, reference it

**Comment by pankrator** [comment]
What do you mean by referencing it?

File: pkg/util/slice/strings.go [hunk 1]
```diff
@@ -63,3 +63,13 @@ func StringsAnyEquals(stringSlice []string, str string) bool {
 	}
 	return false
 }
+
+// StringsAnySuffix returns true if any of the strings in the slice have the given suffix.
+func StringsAnySuffix(stringSlice []string, suffix string) bool {
```
Comment (KirilKabakchiev) [inline pkg/util/slice/strings.go]: what's the reason for adding this util (e.g. where is it needed). Also add a small test for it

Comment (dpanayotov) [reply pkg/util/slice/strings.go]: That's what I meant

Comment (pankrator) [reply pkg/util/slice/strings.go]: It is meant for internal usage.

**Pull request merged**